go.mod

```diff
@@ -1,7 +1,11 @@
 module github.com/Azure/azure-storage-blob-go
 
+go 1.14
+
 require (
 	github.com/Azure/azure-pipeline-go v0.2.1
+	github.com/Azure/go-autorest/autorest/adal v0.8.2
+	github.com/google/uuid v1.1.1
```

Comment:
I have my doubts this will break anything in azcopy, but have you tested compiling azcopy with your modified version of the blob SDK?

Reply:
I have and it does work.